settings.gradle.kts

```
@@ -146,6 +146,7 @@ hideFromDependabot(":instrumentation:apache-httpclient:apache-httpclient-4.0:jav
 hideFromDependabot(":instrumentation:apache-httpclient:apache-httpclient-4.3:library")
 hideFromDependabot(":instrumentation:apache-httpclient:apache-httpclient-4.3:testing")
 hideFromDependabot(":instrumentation:apache-httpclient:apache-httpclient-5.0:javaagent")
+hideFromDependabot(":instrumentation:apache-pulsar:javaagent")
```

for consistency with other instrumentation modules

```suggestion
hideFromDependabot(":instrumentation:apache-pulsar-2.8:javaagent")
```